apps/generator/cli.js

```diff
@@ -84,6 +84,7 @@ program
   })
   .option('-d, --disable-hook [hooks...]', 'disable a specific hook type or hooks from given hook type', disableHooksParser)
   .option('--debug', 'enable more specific errors in the console')
+  .option('--compile', 'compile the template files before rendering')
```

Comment:
after all works remember to remove compile flag from here as we do not enable new flags in local cli